contracts-periphery/src/test/UmbraBatchSend.t.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "ds-test/test.sol";
+import "forge-std/Vm.sol";
+import "forge-std/stdlib.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../UmbraBatchSend.sol";
+
+contract UmbraBatchSendTest is DSTest, stdCheats {
+    using stdStorage for StdStorage;
+    StdStorage stdstore;
+
+    UmbraBatchSend public router = new UmbraBatchSend();
+
+    address umbra = 0xFb2dc580Eed955B528407b4d36FfaFe3da685401;
+    address Dai = 0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735;
+
+    address alice = address(0x202204);
+    address bob = address(0x202205);
+
+    Vm vm = Vm(HEVM_ADDRESS);
+    bytes payload;
+
+    uint public tollCommitment;
+    bytes32 test = "test";
+
+    UmbraBatchSend.SendEth newSendEth;
+    UmbraBatchSend.SendEth[] sendEth;
+    
+    UmbraBatchSend.SendToken newSendToken;
+    UmbraBatchSend.SendToken[] sendToken;
+
+    IERC20 token = IERC20(address(Dai));
+
+    function setUp() public {
+
+        vm.label(alice, "Alice");
+        vm.label(bob, "Bob");
+        vm.label(address(this), "TestContract");
+
+        // address umbraClone = deployCode(umbraArtifact);
+        // emit log_named_address("umbraClone address is ", umbraClone);
+
+         stdstore
+            .target(Dai)
+            .sig(IERC20(token).balanceOf.selector)
+            .with_key(address(this))
+            .checked_write(1000*1e18);
+    }
+
+    function testBatchSendEth() public {
+
+        uint alicePrevBal = alice.balance;
+        uint bobPrevBal = bob.balance;
+
+        uint amount = 1 ether;
+        uint amount2 = 2 ether;
+        uint256 toll = 0;        
+
+        sendEth.push(UmbraBatchSend.SendEth(payable(alice), amount, test, test));
+        sendEth.push(UmbraBatchSend.SendEth(payable(bob), amount2, test, test));
+        router.batchSendEth{value: amount + amount2}(toll, sendEth);
+
+        assertEq(alice.balance, alicePrevBal + amount);
+        assertEq(bob.balance, bobPrevBal + amount2);
+
+    }
+
+    function testFuzz_BatchSendEth(uint8 amount, uint8 amount2, bytes32 testBytes) public {
```

Comment:
Why such small amounts? These are unrealistically small 😛

Also we don't need the concrete tests if we also have the fuzz tests, doesn't seem to add much value